.travis.yml

```diff
@@ -32,6 +31,11 @@ jobs:
     - php: nightly
 
   include:
+    - stage: Test
+      php: nightly
+      before_install:
+        - composer config platform.php 7.4.99
+
```

Comment:
Why this instead of allowing `nightly` to fail?

Reply:
It's not "instead", look at line 31, it's already allowed to fail. But now that you fixed the build, I think we are going to get a green build thanks to this trick, which gives us a way better idea of the state of php 8 support in this repository IMO.